### PR TITLE
update torch dqn, add benchmark results for dqn

### PR DESCRIPTION
**Pull request by swag1ong**

### CODE REVIEW

File: benchmark/torch/dqn/agent.py [hunk 1]
```diff
@@ -12,85 +12,100 @@
...
+        """
+        if obs.ndim == 3:  # if obs is 3 dimensional, we need to expand it to have batch_size = 1
+            obs = np.expand_dims(obs, axis=0)
+
         obs = torch.tensor(obs, dtype=torch.float, device=self.device)
-        pred_q = self.alg.predict(obs)
-        action = pred_q.max(1)[1].item()
-        return action
+        pred_q = self.alg.predict(
+            obs).cpu().numpy().squeeze()  # with torch.no_grad in algorithm
```
Comment (rical730) [inline benchmark/torch/dqn/agent.py]: please use detach() instead of torch.no_grad

**Review by rical730 — APPROVED** [review]
LGTM~
